Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -19,6 +19,7 @@ In order to configure the Ceph storage cluster, at least one of these local stor
 
 * Raw devices (no partitions or formatted filesystems)
 * Raw partitions (no formatted filesystem)
+* Logical Volumes (no formatted filesystem)
```

Comment:
If we are talking about logical volume as in "LVM" let's add it in brackets to it that it is clear what is meant here.

Reply:
How about `LVM Logical Volume (no formatted filesystem)`? I've seen this words many times.

Reply:
Sounds good!